src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -280,7 +280,16 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps, GraphSetting
         id: 'filterSecurity',
         labelText: 'Security',
         isChecked: showSecurity,
-        onChange: toggleGraphSecurity
+        onChange: toggleGraphSecurity,
+        tooltip: (
+          <div style={{ textAlign: 'left' }}>
```

Comment:
I personally think it is a little bit cleaner to declare the tooltip as a `const` rather than inline it.